geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreFactory.scala

```diff
@@ -60,42 +60,41 @@ class AccumuloDataStoreFactory extends DataStoreFactorySpi {
 
     // if no auths are specified we default to the connector auths
     // TODO would it be safer to default to no auths?
-    val auths: List[String] =
-      if (!configuredAuths.isEmpty)
-        configuredAuths.toList
-      else
-        masterAuthsStrings.toList
+    val auths: List[String] = if (configuredAuths.nonEmpty) configuredAuths.toList else masterAuthsStrings.toList
 
     val authorizationsProvider = security.getAuthorizationsProvider(params, auths)
+    val auditProvider = security.getAuditProvider(params).getOrElse {
+      val provider = new ParamsAuditProvider
+      provider.configure(params)
+      provider
+    }
 
     // stats defaults to true if not specified
-    val collectStats = !useMock &&
-        Option(statsParam.lookUp(params)).map(_.toString.toBoolean).forall(_ == true)
-    // caching defaults to false if not specified
-    val caching = Option(cachingParam.lookUp(params)).exists(_.toString.toBoolean)
+    val collectStats = !useMock && Option(statsParam.lookUp(params)).map(_.toString.toBoolean).forall(_ == true)
+
+    val config = AccumuloDataStoreConfig(
+      queryTimeoutParam.lookupOpt[Int](params).map(i => i * 1000L),
```

Comment:
consistency warning...we mix seconds and ms...I feel like we should modify querytimeoutParam in another ticket to be ms or either force a time string e.g.
* 5ms
* 20s
* 100m
* 2h